internal/stop/stop.go

```diff
@@ -23,21 +32,49 @@ func Run(ctx context.Context, fsys afero.Fs) error {
 		}
 	}
 
+	if backup {
+		if err := backupDatabase(ctx, fsys); err != nil {
+			return err
+		}
+	}
+
 	// Stop all services
 	if err := stop(ctx); err != nil {
 		return err
 	}
+	fmt.Println("Stopped " + utils.Aqua("supabase") + " local development setup.")
 
-	// Remove other branches
-	branchDir := filepath.Dir(utils.CurrBranchPath)
-	if err := fsys.RemoveAll(branchDir); err != nil {
-		return err
+	if !backup {
+		// Remove other branches
+		branchDir := filepath.Dir(utils.CurrBranchPath)
+		if err := fsys.RemoveAll(branchDir); err != nil {
+			fmt.Fprintln(os.Stderr, err)
+		}
 	}
```

Comment:
Let's just keep other branches? Otherwise people can't `supabase stop` without backing up without also removing other branches.

Reply:
Current behaviour is

- Without `backup` flag: all branch dirs are deleted
- With `backup` flag: current branch is dumped, other branches are kept as empty dir 

When running `supabase start`, empty branch dirs are deleted.

Reply:
OK, let's just punt the local branching situation